src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -412,12 +412,34 @@ <h3 id="build-queue-queued-heading" jhiTranslate="artemisApp.buildQueue.queuedBu
     }
 </div>
 <div>
-    <div style="display: flex; align-items: center">
-        <h3 id="build-queue-finished-heading" jhiTranslate="artemisApp.buildQueue.finishedBuildJobs" style="margin-right: 20px"></h3>
-        <button class="btn btn-primary" (click)="refresh()">
-            <fa-icon [icon]="faSync" />
-            <span jhiTranslate="metrics.refresh.button"></span>
-        </button>
+    <div class="d-flex justify-content-between">
+        <div style="display: flex; align-items: center">
+            <h3 id="build-queue-finished-heading" jhiTranslate="artemisApp.buildQueue.finishedBuildJobs" style="margin-right: 20px"></h3>
```

Comment:
Is the fixed pixel value `20px` required here or could we use bootstrap's spacing class (`mr-...`)?